packages/flutter/lib/src/scheduler/scheduler.dart

```diff
@@ -18,7 +18,7 @@ double timeDilation = 1.0;
 /// scheduler's epoch. Use timeStamp to determine how far to advance animation
 /// timelines so that all the animations in the system are synchronized to a
 /// common time base.
-typedef void SchedulerCallback(Duration timeStamp);
+typedef void FrameCallback(Duration timeStamp);
```

Comment:
We already have a type like this in the engine:

https://github.com/flutter/engine/blob/master/sky/engine/core/dart/window.dart#L8

We could make the one in dart:ui public and re-export it here, although maybe having a fresh declaration here is better for loose coupling.